docs/local.rst

```diff
@@ -0,0 +1,171 @@
+	sudo crictl start <CONTAINER_ID>
+
+Verify container executed correctly by opening logs:
+
+.. code-block:: bash
+
+	sudo crictl logs <CONTAINER_ID>
+
+
+The expected output is something like the following:
+
+.. code-block:: text
+
+	args: [./test]
+	mounts: 602 548 0:57 / / rw,relatime - overlay overlay rw,lowerdir=/var/run/singularity/containers/fa96e2cdaec1081a8b229fe2d8f64ac80b698b7a07f303629fb60b36abbeec8e/bundle/rootfs,upperdir=/var/run/singularity/containers/fa96e2cdaec1081a8b229fe2d8f64ac80b698b7a07f303629fb60b36abbeec8e/bundle/overlay/upper,workdir=/var/run/singularity/containers/fa96e2cdaec1081a8b229fe2d8f64ac80b698b7a07f303629fb60b36abbeec8e/bundle/overlay/work
+	603 602 0:50 / /proc rw,nosuid,nodev,noexec,relatime - proc proc rw
+	604 602 0:59 / /dev rw,nosuid - tmpfs tmpfs rw,size=65536k,mode=755
+	605 604 0:60 / /dev/pts rw,nosuid,noexec,relatime - devpts devpts rw,gid=5,mode=620,ptmxmode=666
+	606 604 0:61 / /dev/shm rw,nosuid,nodev,noexec,relatime - tmpfs shm rw,size=65536k
+	607 604 0:49 / /dev/mqueue rw,nosuid,nodev,noexec,relatime - mqueue mqueue rw
+	608 602 0:56 / /sys ro,nosuid,nodev,noexec,relatime - sysfs sysfs ro
+	609 602 0:22 /singularity/pods/85d02f45ee7fdf05aa199abafad6b1617fd018b3aacf30883c4724ebb025dac2/hostname /etc/hostname ro,relatime shared:5 - tmpfs tmpfs rw,size=403956k,mode=755
+	610 602 8:1 /var/lib/singularity /mounted1 ro,relatime - ext4 /dev/sda1 rw,errors=remount-ro,data=ordered
+	hostname: networking <nil>
+	pwd: / <nil>
+	content of /
+		     Lrwxrwxrwx        0	.exec -> .singularity.d/actions/exec
+		     Lrwxrwxrwx        0	.run -> .singularity.d/actions/run
+		     Lrwxrwxrwx        0	.shell -> .singularity.d/actions/shell
+		     drwxr-xr-x        0	.singularity.d ->
+		     Lrwxrwxrwx        0	.test -> .singularity.d/actions/test
+		     drwxr-xr-x        0	bin ->
+		     drwxr-xr-x        0	dev ->
+		     Lrwxrwxrwx        0	environment -> .singularity.d/env/90-environment.sh
+		     drwxr-xr-x        0	etc ->
+		     drwxr-xr-x        0	home ->
+		     drwxr-xr-x        0	lib ->
+		     drwxr-xr-x        0	media ->
+		     drwxr-xr-x        0	mnt ->
+		     drwxr-xr-x        0	mounted1 ->
+		     dr-xr-xr-x        0	proc ->
+		     drwx------        0	root ->
+		     drwxr-xr-x        0	run ->
+		     drwxr-xr-x        0	sbin ->
+		     Lrwxrwxrwx        0	singularity -> .singularity.d/runscript
+		     drwxr-xr-x        0	srv ->
+		     dr-xr-xr-x        0	sys ->
+		     -rwxr-xr-x        0	test ->
+		     dtrwxr-xr-x       0	tmp ->
+		     drwxr-xr-x        0	usr ->
+		     drwxr-xr-x        0	var ->
+	uid=0 gid=0 euid=0 egid=0
+	pid=30 ppid=0
+	envs=[LD_LIBRARY_PATH=/.singularity.d/libs SHLVL=1 MY_ANOTHER_VAR=is-awesome PS1=Singularity>  TERM=xterm PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin PWD=/ MY_CUSTOM_VAR=singularity-cri]
+	...
+
+
+Cleanup examples
+----------------
+
+The quickest way to cleanup is simply remove containing pods:
```

Comment:
The easiest/quickest way to do a cleanup is simply **by removing** containing pods: